daml-lf/engine/src/test/scala/com/digitalasset/daml/lf/engine/EngineTest.scala

```diff
@@ -1609,6 +1609,10 @@ object EngineTest {
   private implicit def toName(s: String): Name =
     Name.assertFromString(s)
 
+  private implicit def toGenTransaction(
```

Comment:
I'd rather have this called explicitly. What's the advantage of having this as an implicit conversion? The explicit call looks very brief and understandable.